osu.Game.Rulesets.Mania/ManiaSettingsSubsection.cs

```diff
@@ -37,6 +37,11 @@ private void load()
                     Current = config.GetBindable<double>(ManiaRulesetSetting.ScrollTime),
                     KeyboardStep = 5
                 },
+                new SettingsCheckbox
+                {
+                    LabelText = "Colour-coded notes",
```

Comment:
I'd go with something like "Timing-based note colouring". As it stands it's ambiguous as to what they are colour-coded by (for instance notes can also be colour-coded by the column they are in).